<launch>

	<!-- Load parameters -->
	<rosparam file="$(find osa_communication)/config/bibot.yaml" command="load"/>
	<node name="osa_can_layer_node" pkg="osa_communication" type="osa_can_layer" output="screen" respawn="false">
	
<!--	    <param name="can_device" type="string" value="can0" />

	
	    <param name="controller1_type" type="string" value="EPOS4" />
	    <param name="motor1_type" type="string" value="EC90" />
	    <param name="motor1_inverted" type="bool" value="true" />
	    <param name="mode1" type="string" value="PROFILE_VELOCITY_MODE" />
	    <param name="value1" type="int" value="0" />
	
	    <param name="controller2_type" type="string" value="EPOS4" />
	    <param name="motor2_type" type="string" value="EC90" />
	    <param name="motor2_inverted" type="bool" value="false" />
	    <param name="mode2" type="string" value="PROFILE_VELOCITY_MODE" />
	    <param name="value2" type="int" value="0" />
-->	    
	</node>
	
</launch>
